src/coreclr/nativeaot/Runtime/unix/UnixContext.cpp

```diff
@@ -183,7 +183,43 @@ int unw_get_save_loc(unw_cursor_t*, int, unw_save_loc_t*)
 
 #ifdef HOST_64BIT
 
-#if defined(HOST_ARM64)
+#if defined(HOST_ARM64) and defined(TARGET_FREEBSD)
```

Comment:
A nit - can you please use `&&` instead of `and` to keep the convention used everywhere else?

Reply:
Sure, done.